DataFormats/HLTReco/src/classes_def.xml

```diff
@@ -42,12 +42,16 @@
    <version ClassVersion="10" checksum="2923979599"/>
   </class>
   <class name="trigger::TriggerObjectCollection"/>
-  <class name="trigger::TriggerRefsCollections" ClassVersion="14">
+  <class name="trigger::TriggerRefsCollections" ClassVersion="16">
+   <version ClassVersion="16" checksum="1920377523"/>
+   <version ClassVersion="15" checksum="450679879"/>
```

Comment:
Here you are adding two new versions - but I assume the intermediate ones (`450679879`, `4165317388`, `2824205283`, `948988526`) are actually never used.
Could you remove them and add only one version to each class ?

For example
```suggestion
  <class name="trigger::TriggerRefsCollections" ClassVersion="15">
   <version ClassVersion="15" checksum="1920377523"/>
```

Reply:
Done!